python/doc/examples/data_analysis/statistical_tests/plot_kolmogorov_pvalue.py

```diff
@@ -119,14 +99,15 @@ def drawInTheBounds(vLow, vUp, n_test):
 x = linearSample(KSstat, 0.6, nplot)
 
 # %%
-# Compute the bounds to fill: the lower vertical bound is zero and the upper vertical bound is the KS PDF.
+# Compute the bounds to fill: the lower vertical bound is 0 and the upper vertical bound is the KS PDF.
 
 # %%
-vLow = [[x[i, 0], 0.0] for i in range(nplot)]
-vUp = [[x[i, 0], pKolmogorov.gradient(x[i])[0, 0]] for i in range(nplot)]
+vLow = [[0.0] for i in range(nplot)]
```

Comment:
I think that this should work:
```suggestion
vLow = [0.0] * nplot
```

Reply:
[[0.0]] * nplot works